frontend/src/components/molecules/BlockLink.vue

```diff
@@ -43,7 +42,7 @@ const emitBlur = (newTarget: FocusEvent) => {
 	emit('blur', newTarget)
 }
 
-const handleOnClick = (hash?: string, id?: string) => {
-	if (hash || id) drawer.push({ entityTypeName: 'block', hash: hash || '', id })
+const handleOnClick = (hash: string) => {
+	if (hash) drawer.push({ entityTypeName: 'block', hash: hash })
```

Comment:
If statement should always be true now:
```suggestion
	drawer.push({ entityTypeName: 'block', hash: hash })
```